### ISSUE TITLE
About `access` of keycode modifiers

### ISSUE THREAD

**Issue by tmatsuur** [WordPress/gutenberg#36803]
### Description

In packages/keycodes/src/index.js, the content of `access` is specified as follows.
`access: ( _isApple ) => ( _isApple() ? [ CTRL, ALT ] : [ SHIFT, ALT ] ),`

I have two uncomfortable feelings about the above:
1. `CTRL` is applied when the return value of the isApple function is true.
2. there is no compatibility between [CTRL, ALT] and [SHIFT, ALT] applied by the return value of the isApple function.

By default, `access` is used as a shortcut key to remove a block.
(packages/block-editor/src/components/keyboard-shortcuts/index.js)
`registerShortcut( {
	name: 'core/block-editor/remove',
	category: 'block',
	description: __( 'Remove the selected block(s).' ),
	keyCombination: {
		modifier: 'access',
		character: 'z',
	},
} );
`

If this is the specification, there is no problem, but I felt very uncomfortable, so I posted this.

### Step-by-step reproduction instructions

There is no specific procedure.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by ellatrix** [comment]
I'm not sure I understand your question or concern. Could you rephrase it?

**Comment by tmatsuur** [comment]
Thanks for the comment.

It's been a little while since my first post, so my memory is a little fuzzy.

At the time, I was researching the shortcut key settings and was surprised to find that the key combinations specified in "Access" were different between Mac and Windows.

I wondered if users who use both environments would not be confused by the key differences.

Some shortcut keys were inherited from major applications, and I now wonder if the "access" designation is one of them.

**Comment by ellatrix** [comment]
The difference stems from how shortcuts worked in TinyMCE, and I believe that was done because it's otherwise not possible to find a modifier combination of two modifier keys that works on every platform.

**Comment by tmatsuur** [comment]
So that was the reason.
Thank you for kindly explaining it to me.
